tests/PageObjects/EditSongsPage.ts

```diff
@@ -12,6 +12,10 @@ export class EditSongsPagePO {
     await this.page.locator(`[data-test="hide-song"][data-song="${songID}"]`).click();
   }
 
+  public async expectHideSongNotToBeVisible(songID: string) {
+    await expect(this.page.locator(`[data-test="hide-song"][data-song="${songID}"]`)).not.toBeVisible();
```

Comment:
it should use `restore-song` data test in the assertion (expect it to be visible). The `hide-song` button could not be visible for various reasons, e.g. the song is not visible on the list at all